ssl/t1_lib.c

```diff
@@ -992,13 +992,22 @@ static const uint16_t tls12_sigalgs[] = {
     TLSEXT_SIGALG_rsa_pkcs1_sha384,
     TLSEXT_SIGALG_rsa_pkcs1_sha512,
 
+/* Presense of OPENSSL_CAPIENG_LIMIT_SIGALGS removes sha-224 signature
```

Comment:
NITS: (note spelling error) and /* */ on separate lines
/*
 * The presence of ....
 * ....
 *
 */

Reply:
Fixed.